accepted/0000-caasp-nodes.md

```diff
@@ -0,0 +1,277 @@
+- Feature Name: CaaSP nodes action locking
+- Start Date: 2019-10-28
+- RFC PR: TBD-change filename
+
+# Summary
+[summary]: #summary
+
+This RFC describes the special handling that Uyuni/SUSE Manager must implement to manage CaaS Platform nodes.
+
+# Motivation
+[motivation]: #motivation
+
+<!-- - Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+Describe the problem you are trying to solve, and its constraints, without coupling them too closely to the solution you have in mind. If this RFC is not accepted, the motivation can be used to develop alternative solutions. -->
+
+Any node of a CaaS Platform cluster can be registered to Uyuni/SUSE Manager.
+Registering a CaaS Platform node to Uyuni/SUSE Manager brings the following notable advantages:
+
+- The user can see the patch level status of the CaaS Platform nodes (at the packages level, not at the container level - the latter is called "[container staging](https://confluence.suse.com/display/~dmacvicar/Container+Staging+Breakout)" and it is outside of the scope of this RFC)
+- The user can perform configuration management operations using Salt from Uyuni/SUSE Manager
+- The user can assign a different set of channels (with channel staging and/or CLM filters) to different clusters and manually control from Uyuni/SUSE Manager which version of CaaS Platform gets automatically installed in its CaaSP clusters<sup>1</sup>.
+
+However, if the user does some specific actions on any node of the cluster via Uyuni/SUSE Manager, the cluster may result temporarily or permanently unusable or broken.
```

Comment:
I am confused a bit now: is this RFC exclusively about actions (in the Uyuni concept sense of the word) or are we thinking about both actions and Salt commands?

IOW, are we happy to block some buttons in the UI for now, or do we want to prevent also users calling Salt directly, or writing their own states, from breaking the cluster?

Note that the answer can very well be "we want to do both, but this RFC is just the first step so it only concentrates on the former for now"

Reply:
The first step is to limit the damage that a user can trigger via Uyuni/SUSE Manager.
Secondly, limit the damage via Salt.
Does it make sense?

Reply:
It does, and it may make sense to treat the latter as a "Next Step" or even separate RFC.